rtlsdr_wsprd.c

```diff
@@ -794,10 +805,20 @@ int main(int argc, char **argv) {
     uint32_t nLoop = 0;
 
     if (argc <= 1)
-        usage();
+        usage(stderr, 1);
 
-    while ((opt = getopt(argc, argv, "f:c:l:g:ao:p:u:d:n:i:tw:r:HQS")) != -1) {
+    while ((opt = getopt_long(argc, argv, short_options, long_options, &option_index)) != -1) {
         switch (opt) {
+            case 0:
+                switch (option_index) {
+                    case 0:  // --help
+                        usage(stdout, 0);
+                        break;
+                    case 1:  // --version
+                        printf("rtlsdr_wsprd 0.5.3\n");
```

Comment:
Yep, adding a global variable for the version number makes sense (cf. your comment in the MR)